common/lib/api-client/request-headers.js

```diff
@@ -13,7 +13,7 @@ module.exports = (req, format = 'application/vnd.api+json') => {
     'Idempotency-Key': uuid.v4(),
   }
 
-  if (req && req.user) {
+  if (req && req.user && req.user.username) {
```

Comment:
And, before I even started typing I remembered "testcafe and optional chaining" :joy: